### PR TITLE
Added Update-VSTeamGitRepositoryDefaultBranch

### PR DESCRIPTION
**Pull request by rbleattler**
# PR Summary

- Added Update-VSTeamGitRepositoryDefaultBranch to allow for changing the default branch of a repository
- Fixed bad link in README.md
- Bumped manifest version
- Cleaned up comments/debug messages
- Proposed changes from original PR : MethodsAndPractices/vsteam/pull/474

## PR Checklist

- [X] [Write Help](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-help)
- [X] [Write Unit Test](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#write-unit-test)
- [X] [Update CHANGELOG.md](https://github.com/DarqueWarrior/vsteam/blob/master/.github/CONTRIBUTING.md#update-changelogmd)

### CODE REVIEW

**Comment by rbleattler** [comment]
@SebastianSchuetze do I need to re-submit with SupportsShouldProcess implemented?

**Comment by SebastianSchuetze** [comment]
What do you mean resubmit? You can just keep editing your branch until everything is fine. You didn't need to close the PR and reopen one.

Just commit here until everything is fine. At the end we do a squash merge anyways.

**Comment by rbleattler** [comment]
Thanks, I learn something new every day. I'll try to get this updated on Monday.

**Comment by rbleattler** [comment]
Updated Update-GitVSTeamGitRepositoryDefaultBranch with SupportsShouldProcess / -WhatIf support.

**Comment by rbleattler** [comment]
I believe everything that needed to be fixed has been fixed, including adding support for SupportsShouldProcess / -WhatIf

File: Tests/function/tests/_testInitialize.ps1 [hunk 1]
```diff
@@ -11,8 +11,18 @@ Import-Module SHiPS
 
 $baseFolder = "$PSScriptRoot/../../.."
 $sampleFiles = "$PSScriptRoot/../../SampleFiles"
-      
-Add-Type -Path "$baseFolder/dist/bin/vsteam-lib.dll"
+
+# Changed to only import this type if the dll is missing and exists in the path, since PS throws an exception when trying to import a dll already in memory
+$vsTeamDllPath = "$baseFolder/dist/bin/vsteam-lib.dll"
```
Comment (SebastianSchuetze) [inline Tests/function/tests/_testInitialize.ps1:16]: thanks for this addition! :)

**Review by SebastianSchuetze — DISMISSED** [review]

**Comment by SebastianSchuetze** [comment]
I removed some lines in the changelog because it is not interesting if there are typo changes. Only real changes for somebody using the modules is interesting. E.g. bug fixes or new features in cmdlets or the documentation of the cmdlets themselve.

approved and merged. Thanks very much!!

**Pull request merged**